docs/user/openstack/README.md

```diff
@@ -44,6 +47,69 @@ enough to store the ignition config files, so they are served by swift instead.
 * You may need to increase the security group related quotas from their default
   values. For example (as an OpenStack admin) `openstack quota set --secgroups 100 --secgroup-rules 1000 <project>`
 
+
+* A file `install-config.yaml` is required. It defines your cluster topology. Save it because a failed installation will delete it. It looks like this:
+
+```
+apiVersion: v1
+baseDomain: example.com
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.master
+  replicas: 3
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: m1.master
+  replicas: 1
```

Comment:
Please set the `replicas` to `3`. That's the default and while changing this value at install-time makes sense, I'd like to display the recommended value here.